front-end/virtual_pet/lib/widgets/todo_tile.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:virtual_pet/widgets/todo_item.dart';
+
+// Todo item tile widget
+class TodoItemTile extends StatefulWidget {
+  final Todo todo;
+  final Function (Todo todo) onChecked;
```

Comment:
Can we remove the ` ` between the end of the `Function` type and `(` so it is consistent with the one below and with function definitions/calls more generally?